Sources/MountebankSwift/Models/Stub/Response/Proxy.swift

```diff
@@ -57,5 +73,51 @@ struct ProxyParameters {
     /// Shared passphrase used for a single private key
     let passphrase: String?
     /// Key-value pairs of headers to inject into the proxied request.
-    let injectHeaders: [String: String]
+    let injectHeaders: [String: JSON]
+}
+
+public struct PredicateGenerator: Codable, Equatable {
```

Comment:
- [ ] Move to own file

Reply:
Maybe create the same kind of `enum` as `Predicate` for this config?

Reply:
Updated